packages/target-tauri/src-tauri/src/stickers.rs

```diff
@@ -0,0 +1,112 @@
+use anyhow::Context;
+use log::error;
+use percent_encoding::percent_decode_str;
+use tauri::{Manager, UriSchemeContext, UriSchemeResponder};
+use tokio::fs;
+
+use crate::state::deltachat::DeltaChatAppState;
+
+pub(crate) fn delta_stickers_protocol<R: tauri::Runtime>(
+    ctx: UriSchemeContext<'_, R>,
+    request: http::Request<Vec<u8>>,
+    responder: UriSchemeResponder,
+) {
+    // info!("dcsticker {}", request.uri());
+
+    // URI format is dcsticker://<account folder name>/<sticker pack>/<sticker filename>
+
+    let app_state_deltachat = {
+        ctx.app_handle()
+            .state::<DeltaChatAppState>()
+            .deltachat
+            .clone()
+    };
+
+    tauri::async_runtime::spawn(async move {
+        // workaround for not yet available try_blocks feature
+        // https://doc.rust-lang.org/beta/unstable-book/language-features/try-blocks.html
+        let result: anyhow::Result<()> = async {
+            // parse url (account folder name, sticker pack folder and sticker filename)
+            let parsed = {
+                let mut splited = request.uri().path().split('/');
+                #[cfg(not(any(target_os = "windows", target_os = "android")))]
+                {
+                    (request.uri().host(), splited.nth(1), splited.next())
+                }
+                #[cfg(any(target_os = "windows", target_os = "android"))]
+                {
+                    (splited.nth(1), splited.next(), splited.next())
+                }
+            };
```

Comment:
This parse code here together with the one in `blobs.rs` would be nice to put together somewhere in some utility functions and write some tests for it.

Reply:
tests sound good, but I doubt anything will break here as the code is still very simple.